twill-api/src/main/java/org/apache/twill/api/TwillRunResources.java

```diff
@@ -59,6 +61,16 @@
   /**
    * @return the enabled log level for the container where the runnable is running in.
    */
+  @Deprecated
```

Comment:
Why deprecating a method, also add a `@deprecated` in the javadoc section to tell what's the alternative. E.g.

```
/**
 * ....
 * @deprecated Use {@link #getRootLogLevel()} instead.
 */